src/NexusMods.App/Services.cs

```diff
@@ -107,12 +103,7 @@ private static IServiceCollection AddSupportedGames(this IServiceCollection serv
     {
         if (experimentalSettings is { EnableAllGames: true })
         {
-            Games.BethesdaGameStudios.Services.AddBethesdaGameStudios(services);
             Games.RedEngine.Services.AddRedEngineGames(services);
-            Games.DarkestDungeon.Services.AddDarkestDungeon(services);
-            Games.BladeAndSorcery.Services.AddBladeAndSorcery(services);
-            Games.Sifu.Services.AddSifu(services);
-            Games.MountAndBlade2Bannerlord.ServicesExtensions.AddMountAndBladeBannerlord(services);
         }
```

Comment:
The setting `EnableAllGames` can be removed as well.

Reply:
We don't officially have cyberpunk support in yet, and we have yet to find a person to drive that work this sprint, so I'll keep it around for now.

Reply:
I'll get FOMODs added back in